examples/backends/vllm/launch/lora/xpu/multimodal/README.md

```
@@ -0,0 +1,239 @@
+|---|---|
+| `Qwen/Qwen3-VL-2B-Instruct` | Default. Uses `--gpu-memory-utilization 0.78` for single-XPU testing. |
+| `Qwen/Qwen2.5-VL-7B-Instruct` | Higher quality, needs 24 GB+ VRAM. |
+| `llava-hf/llava-1.5-7b-hf` | LLaVA architecture, 4096 max context. |
+
+## LoRA Management API
+
+All management endpoints are served on the system port (default 8081).
+
+### Load a LoRA
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)